pyclean/modern.py

```diff
@@ -88,9 +88,25 @@ def descend_and_clean_bytecode(directory):
             if child.suffix in ['.pyc', '.pyo']:
                 Runner.unlink(child)
         elif child.is_dir():
-            descend_and_clean_bytecode(child)
+            if child.name == '.pytest_cache' and clean_test_cache:
```

Comment:
We can probably make this more generic by maintaining a list of folders to remove. This will not be the only folder to delete.